### ISSUE TITLE
Hand/RCD made airlock let gas pass

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#10009]
(WEB REPORT BY: pwirby REMOTE: 172.93.109.202:7777)
#### Revision (Should be above if you're viewing this from ingame!)

I made an airlock using the RCD and purged the gas out of one of the adjacent room and both of the rooms were emptied. I had the same issue with airlocks made with metal sheets, wires and airlock electronics.
#### General description of the issue

The hand/RCD made airlock let gas pass throught them.
#### What you expected to happen

Airlocks block gas from going out of the room.
#### What actually happened

Both room were emptied.
#### Steps to reproduce if possible

Make a room with an airlock that you made (with metal sheets or RCD),  build some pipe with an unary vent in the room you want to empty and an injector leading to space, set the unary vent to purge, turn both on (using AAC) and enjoy the airlock blocking gas not working.

**Comment by FudgePucker** [comment]
> gaz

**Comment by PJB3005** [comment]
Fixed the grammar.

**Comment by clusterfack** [comment]
Likely it needs update nearby tiles, strange though because that should be in new()

**Comment by Coxswain-Navigator** [comment]
Just had this happen to me when doing a clown gimmick, i surrounded myself with bolted airlocks and turned on some gas miners, the whole room then proceeded to flood with gas completely ignoring the airlocks i hand-crafted.

**Comment by Exxion** [comment]
Has this happened recently? If so, does it persist if the door is opened and closed again?

**Comment by Shadowmech88** [comment]
@Exxion 
Tested locally. I created a 3x3 square of airlocks with an RCD, and put a gas miner in the middle. The airlocks did not prevent the gas miner from over-pressurizing the room. Then, I opened all of the airlocks and closed them all again, and after returning the room's pressure to normal, it did not keep rising. I then opened one of the airlocks, and the room's pressure began rising again.

This suggests that opening and closing the airlock does cause it to update its air-tightness.
